### PR TITLE
fix: Milestone styling in both simple and detail view

### PR DESCRIPTION
**Pull request by whizzzkid**
Closes: #123 

In this PR:
- Fixed margins between horizontal and vertical stacked milestones. [Demo](https://starmaps-git-fix-milestone-styleing-ipfs-ignite.vercel.app/roadmap/github.com/pln-planning-tools/roadmap-test/issues/5#simple)
- Fixed padding in group milestones in detail view. [Demo](https://starmaps-git-fix-milestone-styleing-ipfs-ignite.vercel.app/roadmap/github.com/pln-planning-tools/roadmap-test/issues/5#detail)
- Renamed `GroupItem` -> `GroupHeader`

### CODE REVIEW

**Comment by SgtPooki** [comment]
only callout is that milestone's border-radius seems to be gone. See starmaps.app:
image

File: components/roadmap-grid/Roadmap.module.css [hunk 1]
```diff
@@ -51,8 +51,7 @@
   padding: 5px;
   padding-left: 10px;
   padding-right: 10px;
-  margin-bottom: 4px;
-  border-radius: 8px;
```
Comment (SgtPooki) [inline components/roadmap-grid/Roadmap.module.css:55]: we want to keep this

File: components/roadmap-grid/RoadmapDetailedView.tsx [hunk 2]
```diff
@@ -161,13 +161,12 @@ export function RoadmapDetailed({
...
-              <GroupWrapper key={index} >
-                <GroupItem group={group} />
+              <><GroupHeader group={group} /><GroupWrapper key={index}>
                 {!!group.items &&
                   _.sortBy(group.items, ['title']).map((item, index) => {
-                    return <GridRow key={index} timeScaler={globalTimeScaler} milestone={item} index={index} timelineTicks={ticks} numGridCols={numGridCols} numHeaderItems={numHeaderTicks}/>;
+                    return <GridRow key={index} timeScaler={globalTimeScaler} milestone={item} index={index} timelineTicks={ticks} numGridCols={numGridCols} numHeaderItems={numHeaderTicks} />;
                   })}
-              </GroupWrapper>
+              </GroupWrapper></>
```
Comment (SgtPooki) [inline components/roadmap-grid/RoadmapDetailedView.tsx:169]: nit: prefer fragments on individual lines so it's easier to see

**Review by SgtPooki — CHANGES_REQUESTED** [review]

**Comment by SgtPooki** [comment]
also this example seems to show that milestone ticks are off :|

**Comment by juliaxbow** [comment]
> - Fixed margins between horizontal and vertical stacked milestones. [Demo](https://starmaps-git-fix-milestone-styleing-ipfs-ignite.vercel.app/roadmap/github.com/pln-planning-tools/roadmap-test/issues/5#simple)

LGTM

> - Fixed padding in group milestones in detail view. [Demo](https://starmaps-git-fix-milestone-styleing-ipfs-ignite.vercel.app/roadmap/github.com/pln-planning-tools/roadmap-test/issues/5#detail)

In the detailed view can we double the current height of the top/bottom margins? So it should look like the Ubiquitous Clients section in the screenshot below (doubled; mockup) and not the Privacy section (existing; from preview)

Screen Shot 2022-11-22 at 4 12 35 PM

**Review by SgtPooki — APPROVED** [review]
im good with it if Julia is =)

**Pull request merged**